tools/namefixer.py

```diff
@@ -698,6 +698,8 @@
     "globalCtx->envCtx.unk_DC": "play->envCtx.lightBlend",
     "globalCtx->interfaceCtx.unk_21E": "play->interfaceCtx.bButtonDoAction",
 
+    "play->actorCtx.flags": "play->actorCtx.sceneFlags",
+    "play->actorCtx.unk5": "play->actorCtx.flags",
```

Comment:
This produces a recursive problem... Running `./tools/namefixer.py` should warn about it

Reply:
I wasn't too sure about this, should I just leave this out then?

Reply:
I think you could leave them commented out